### PR TITLE
kcc: Add version 5.5.1

### PR DESCRIPTION
**Pull request by davidxuang**
Adding [Kindle Comic Converter](https://github.com/ciromattia/kcc) (not the Amazon Kindle Comic Creator).

### CODE REVIEW

File: bucket/kcc.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                    "Kindle Comic Converter"
+                ]
+            ]
+        }
+    },
+    "innosetup": true,
+    "depends": "extras/kindlegen",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "http://kcc.iosphe.re/Windows/#/dl.exe"
```
Comment (Ash258) [inline bucket/kcc.json]:
```suggestion
    "architecture": {
        "64bit": {
            "url": "https://kcc.iosphe.re/Windows/KindleComicConverter_win_$version.exe",
        }
    }
```

File: bucket/kcc.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "5.5.1",
+    "description": "Comic and manga converter for ebook readers.",
+    "homepage": "https://github.com/ciromattia/kcc",
+    "license": "ISC",
+    "architecture": {
+        "64bit": {
+            "url": "http://kcc.iosphe.re/Windows/#/dl.exe",
```
Comment (Ash258) [inline bucket/kcc.json]:
```suggestion
            "url": "https://kcc.iosphe.re/Windows/KindleComicConverter_win_5.5.1.exe",
```

File: bucket/kcc.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                    "Kindle Comic Converter"
+                ]
+            ]
+        }
+    },
+    "innosetup": true,
+    "depends": "extras/kindlegen",
+    "checkver": "github",
+    "autoupdate": {
+    "architecture": {
```
Comment (Ash258) [inline bucket/kcc.json]: I missed one indentation level. Indent it correctly please.

**Comment by davidxuang** [comment]
By the way, is it okay to use such a short identifier? It is the name of repo, but seems easy to cause conflicts (`kcc` is short for `Kindle Comic Creator` as well).

**Comment by Ash258** [comment]
I really do not mind naming. And as I am not user of this application and never heard of it, I cannot tell if vendors are propagating this application with shortname or fullname

**Comment by davidxuang** [comment]
Thanks for replying, maybe I should just keep the current name since it's used in AUR.

**Pull request merged**